### ISSUE TITLE
 v0.10.2 is broken on python 2.7.8 Windows platform

### ISSUE THREAD

**Issue by fengxx** [httplib2/httplib2#44]
```
import httplib2
conn=httplib2.Http(timeout=60,disable_ssl_certificate_validation=True)
conn.request("https://www.google.com")

>>> conn=httplib2.Http(timeout=60,disable_ssl_certificate_validation=True)
>>> conn.request("https://www.google.com")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\ven\lib\site-packages\httplib2\__init__.py", line 1649
, in request
    (response, content) = self._request(conn, authority, uri, request_uri, metho
d, body, headers, redirections, cachekey)
  File "C:\ven\lib\site-packages\httplib2\__init__.py", line 1389
, in _request
    (response, content) = self._conn_request(conn, request_uri, method, body, he
aders)
  File "C:\ven\lib\site-packages\httplib2\__init__.py", line 1310
, in _conn_request
    conn.request(method, request_uri, body, headers)
  File "C:\Python27\Lib\httplib.py", line 995, in request
    self._send_request(method, url, body, headers)
  File "C:\Python27\Lib\httplib.py", line 1029, in _send_request
    self.endheaders(body)
  File "C:\Python27\Lib\httplib.py", line 991, in endheaders
    self._send_output(message_body)
  File "C:\Python27\Lib\httplib.py", line 844, in _send_output
    self.send(msg)
  File "C:\Python27\Lib\httplib.py", line 820, in send
    self.sock.sendall(data)
AttributeError: sendall

```

**Comment by temoto** [comment]
Does the same code work with 0.9.2?

**Comment by temoto** [comment]
I think I located the problem. Please, try again with this patch

`pip install https://github.com/eventlet/eventlet/archive/issue-44.zip`

**Comment by fengxx** [comment]
0.9.2 works fine, and the fix works too. Thanks!

**Comment by fengxx** [comment]
checked httplib.FakeSocket, it is just 
```

    def FakeSocket (sock, sslobj):
        warnings.warn("FakeSocket is deprecated, and won't be in 3.x.  " +
                      "Use the result of ssl.wrap_socket() directly instead.",
                      DeprecationWarning, stacklevel=2)
        return sslobj

```

I think it is the same issue as `https://github.com/httplib2/httplib2/issues/45`, v0.10.3 works fine, no need to patch FakeSocket.

**Comment by temoto** [comment]
Resolved in #45
